validate/__main__.py

```diff
@@ -46,6 +46,8 @@ def main():
                 validate_dir(path, input_type)
             elif input_type in CHECKSUM_GEN_TYPES:
                 create_checksum_file(path, input_type)
+        except FileNotFoundError as err:
+            print_index_warning(path)
```

Comment:
We may want to include the message from L23 in bam.py; that message seems to be more specific/detailed.